fastly.yaml

```diff
@@ -0,0 +1,4 @@
+---
+vhost:
```

Comment:
I know you've inherited `vhost` from the existing script, but since this is PR#1 that could quickly be changed to something else before it's in use by anyone else... ;)